### PR TITLE
[2203] Add Get-AzsComputeScaleUnit command, docs, and examples

### PR DESCRIPTION
**Pull request by travist13**

### CODE REVIEW

File: src/Azs.Compute.Admin/Azs.Compute.Admin.psd1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+@{
```
Comment (bganapa) [inline src/Azs.Compute.Admin/Azs.Compute.Admin.psd1:1]: this file is generated, pls dont check in

File: src/Azs.Compute.Admin/readme.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ This module was primarily generated via [AutoRest](https://github.com/Azure/auto
 ## Module Requirements
 - [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
 
+## Module Requirements
```
Comment (bganapa) [inline src/Azs.Compute.Admin/readme.md:22]: This part is repeated twice

**Review by TheOnlyWei — DISMISSED** [review]

File: src/Azs.Compute.Admin/readme.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ metadata:
 
 ### PSD1 metadata changes
 subject-prefix: ''
-module-version: 1.1.0
```
Comment (bganapa) [inline src/Azs.Compute.Admin/readme.md:58]: version to 1.2.0 for cmdlet addition(minor version update), we typically update the revision (1.1.1) alone for bug fixes, major version for breaking changes (2.1.0)

File: src/Azs.Compute.Admin/readme.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ metadata:
...
+module-version: 1.1.1
 service-name: ComputeAdmin
```
Comment (bganapa) [inline src/Azs.Compute.Admin/readme.md:59]: Please update the change log as well https://github.com/Azure/azurestack-powershell/blob/dev/src/changelog.md
Code review checklist is at https://github.com/Azure/azurestack-powershell/wiki/Code-Review-Check-List

**Review by bganapa — APPROVED** [review]

**Review by TheOnlyWei — APPROVED** [review]

**Pull request merged**